src/main/java/liquibase/changelog.xml

```diff
@@ -13,6 +13,7 @@
     <include file="classpath:/liquibase/addConvertedFromMeasureColumn.xml"/>
     <include file="classpath:/liquibase/addMeasureDetailsReferenceType.xml"/>
     <include file="classpath:/liquibase/addFhirDTlFeatureFlags.xml"/>
+    <include file="classpath:/liquibase/addFhirBonnieFeatureFlag.xml"/>
```

Comment:
Maybe not critical in that particular case. But I would suggest appending new change-sets at the end of the list.